docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -1,9 +1,293 @@
+      },
+  "2": {
+        "Code": "INR",
+        "Currency": "Indian Rupee"
+      },
+  "3": {
+        "Code": "MXN",
+        "Currency": "Mexican Peso"
+      },
+  "4": {
+        "Code": "GBP",
+        "Currency": "Pounds Sterling"
+      },
+  "5": {
+        "Code": "TRY",
+        "Currency": "Turkish Lira"
+      },
+  "6": {
+        "Code": "USD",
+        "Currency": "United States Dollar"
+      },
+}
+
+```
+. As before, from the command line change into the `$HOME/.local/share/hz-cli` directory. This time execute a `put-all` command to write the data from the JSON file to a new map called `currency`.
++
+```bash
+hzc map put-all -n currency --json-entry currencycodes.json
+```
++
+The `-n` flag passes the map name to your cluster.
+
+. Do a quick check on your cluster to make sure that your data has been written successfully.
+.	Open the dashboard of the development cluster and click *Management Center*.
+.	Go to *Storage* > *Maps*. You'll see that your cluster has a map called `currency` with six entries. 
+
+
+[[step-6-query-map]]
+== Step 6. Query Map Data
+You can use SQL to query the data in your `currency` map.
+
+.	Start by creating a mapping to the `currency` map.
+
++
+```bash
+hzc sql "CREATE MAPPING currency (__key VARCHAR, Code VARCHAR, Currency VARCHAR) TYPE IMap OPTIONS('keyFormat'='varchar', 'valueFormat'='json-flat')"
+```
+The SQL mapping statement does a number of things:
+
+** Adds column headings for currencies and codes
+** Creates a SQL connection to the map
+** Tells Hazelcast how to serialize and deserialize the keys and values.
+
+. Try running some simple queries against the `currency` map. For example, this query returns all data in the map and orders it by the currency code.  
++
+```bash
+hzc sql "SELECT * FROM currency ORDER BY Code"
+```
++
+The results look like this:
+
++
+[source,shell]
+----
++------------+--------------------+--------------------+
+|__key       |Code                |Currency            |
++------------+--------------------+--------------------+
+|1           |CAD                 |Canadian Dollar     |
+|4           |GBP                 |Pounds Sterling     |
+|2           |INR                 |Indian Rupee        |
+|3           |MXN                 |Mexican Peso        |
+|5           |TRY                 |Turkish Lira        |
+|6           |USD                 |United States Dollar|
++------------+--------------------+--------------------+
+
+----
+
+[[step-7-automate]]
+== Step 7. Automate Actions
+When you're ready, combine the commands that you've learned about so far into a shell script and run them from the command line. 
+
+The script first writes the currency data to a new map called `currencydata` on your development server, queries it and then switches to your production cluster to perform the same actions.
+
+[tabs] 
+==== 
+Linux:: 
++ 
+--
+. Copy the following commands into a shell script.
++
+.myscript.sh
+[source,bash]
+----
+#!/usr/bin/bash
```

Comment:
```suggestion
#!/bin/bash
```

I had to use this shebang on my Mac. Maybe we need to provide another tab for Mac?

Reply:
Added a new tab as suggested.